composer.json

```diff
@@ -2,40 +2,46 @@
     "name": "olcs/olcs-common",
     "description": "Common library for the OLCS Project",
     "type": "library",
-    "repositories": [
-        {
-            "packagist": false
-        },
-        {
-            "type": "composer",
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/Packagist-Proxy",
-            "only": [
-                "olcs/olcs-logging",
-                "olcs/olcs-utils"
-            ]
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/OLCS",
-            "type": "composer"
-        },
-        {
-            "url": "https://nexus.olcs.dev-dvsacloud.uk/repository/Packagist-Proxy",
-            "type": "composer"
-        }
-    ],
     "require": {
-        "php": ">=7.4",
-        "olcs/olcs-laminas": "~2.5.0 || ~3.0.0"
+        "php": "^7.4",
+        "laminas/laminas-authentication": "^2.6",
+        "laminas/laminas-form": "^2.13",
+        "laminas/laminas-servicemanager": "^2.7",
+        "laminas/laminas-cache": "^2.8",
+        "laminas/laminas-eventmanager": "^2.6",
+        "laminas/laminas-http": "^2.8",
+        "laminas/laminas-i18n": "^2.14",
+        "laminas/laminas-modulemanager": "^2.9",
+        "laminas/laminas-mvc": "^2.7",
+        "laminas/laminas-view": "^2.11",
+        "laminas/laminas-validator": "^2.11",
+        "olcs/olcs-utils": "^5.0",
+        "dvsa/authentication-cognito": "^2.0",
+        "laminas/laminas-console": "^2.7",
```

Comment:
Does this library depend on this?

Reply:
yes validator and utils been used in source code

Reply:
Sorry Sana, these comments only relate the the line they are attached to, so in this case, line 20.

Reply:
ya console been removed ,

Reply:
Why was `dvsa/authentication-cognito` removed?